tools/wptrunner/wptrunner/browsers/firefox.py

```diff
@@ -251,9 +251,11 @@ def start(self, group_metadata=None, **kwargs):
         if self.ca_certificate_path is not None:
             self.setup_ssl()
 
+        args = self.binary_args[:] if self.binary_args else []
+        args += [cmd_arg("marionette"), "about:blank"]
```

Comment:
I think we possibly want to check for duplicate arguments. It doesn't make much difference, but is a little cleaner. 
```
If not any(item.strip("-/") == "marionette" for item in args):
  args += "--marionette"
```

Just appending `about:blank` is probably fine, although we might worry about args not starting with `--`, `-` or `/`, which would be interpreted as URLs and could in some future hypothetically start opening multiple tabs.

Reply:
Since we have a utility that's designated for creation of CLI arguments, I'd rather incorporate that:

    marionette_flag = cmd_arg("marionette")
    if not any(item == marionette_flag for item in args):
        args.append(marionette_flag)

...but I don't think this discussion should delay the bug fix. I'm also not sure how to relate the new behavior to the fix in the commit message. What do you think about deferring this for a follow-up patch?

Reply:
That doesn't work because multiple variants are supported. The Firefox command line parser is something quite… special.

I'm OK on this not delaying the fix though.